db/data/npq_contracts/fake-2021.csv

```diff
@@ -1,78 +1,78 @@
-LLSE,2021,npq-leading-teaching-development,1038,938,50,FALSE,0.0
-LLSE,2021,npq-senior-leadership,1039,939,51,FALSE,0.0
-LLSE,2021,npq-headship,1040,940,52,FALSE,0.0
-LLSE,2021,npq-executive-leadership,1041,941,53,FALSE,0.0
-LLSE,2021,npq-leading-literacy,1042,942,54,FALSE,0.0
-LLSE,2021,npq-early-years-leadership,1043,943,55,FALSE,0.0
-Teach First,2021,npq-leading-teaching,1044,944,56,FALSE,0.0
-Teach First,2021,npq-leading-behaviour-culture,1045,945,57,FALSE,0.0
-Teach First,2021,npq-leading-teaching-development,1046,946,58,FALSE,0.0
-Teach First,2021,npq-senior-leadership,1047,947,59,FALSE,0.0
-Teach First,2021,npq-headship,1048,948,60,FALSE,0.0
-Teach First,2021,npq-executive-leadership,1049,949,61,FALSE,0.0
-Teach First,2021,npq-leading-literacy,1050,950,62,FALSE,0.0
-Teach First,2021,npq-early-years-leadership,1051,951,63,FALSE,0.0
-Teacher Development Trust,2021,npq-leading-teaching,1052,952,64,FALSE,0.0
-Teacher Development Trust,2021,npq-leading-behaviour-culture,1053,953,65,FALSE,0.0
-Teacher Development Trust,2021,npq-leading-teaching-development,1054,954,66,FALSE,0.0
-Teacher Development Trust,2021,npq-senior-leadership,1055,955,67,FALSE,0.0
-Teacher Development Trust,2021,npq-headship,1056,956,68,FALSE,0.0
-Teacher Development Trust,2021,npq-executive-leadership,1057,957,69,FALSE,0.0
-Teacher Development Trust,2021,npq-leading-literacy,1058,958,70,FALSE,0.0
-Teacher Development Trust,2021,npq-early-years-leadership,1059,959,71,FALSE,0.0
-UCL Institute of Education,2021,npq-leading-teaching,1060,960,72,FALSE,0.0
-UCL Institute of Education,2021,npq-leading-behaviour-culture,1061,961,73,FALSE,0.0
-UCL Institute of Education,2021,npq-leading-teaching-development,1062,962,74,FALSE,0.0
-UCL Institute of Education,2021,npq-senior-leadership,1063,963,75,FALSE,0.0
-UCL Institute of Education,2021,npq-headship,1064,964,76,FALSE,0.0
-UCL Institute of Education,2021,npq-executive-leadership,1065,965,77,FALSE,0.0
-UCL Institute of Education,2021,npq-leading-literacy,1066,966,78,FALSE,0.0
-UCL Institute of Education,2021,npq-early-years-leadership,1067,967,79,FALSE,0.0
-Ambition Institute,2021,npq-early-headship-coaching-offer,300,800,0,FALSE,0.0
-Best Practice Network,2021,npq-early-headship-coaching-offer,300,800,0,FALSE,0.0
-Church of England,2021,npq-early-headship-coaching-offer,300,800,0,FALSE,0.0
-Education Development Trust,2021,npq-early-headship-coaching-offer,300,800,0,FALSE,0.0
-School-Led Network,2021,npq-early-headship-coaching-offer,300,800,0,FALSE,0.0
-LLSE,2021,npq-early-headship-coaching-offer,300,800,0,FALSE,0.0
-Teach First,2021,npq-early-headship-coaching-offer,300,800,0,FALSE,0.0
-Teacher Development Trust,2021,npq-early-headship-coaching-offer,300,800,0,FALSE,0.0
-UCL Institute of Education,2021,npq-early-headship-coaching-offer,300,800,0,FALSE,0.0
+provider_name,cohort_year,course_identifier,recruitment_target,per_participant,service_fee_installments,special_course,monthly_service_fee,funding_cap
```

Comment:
could we set this to `nil` rather than zero? just so we can tell the difference between 2024 and the other cohorts

Reply:
Hi @cwrw the value `0.0` belongs to `monthly_service_fee` column, I just added a comma in the end so `funding_cap` is populated with `nil`.